resources/chef_ingredient.rb

```diff
@@ -20,34 +20,35 @@
 
 default_action :install
 
-property :product_name, kind_of: String, name_attribute: true
-property :config, kind_of: String
+property :product_name, String, name_attribute: true
+property :config, String
 
 # Determine what version to install from which channel
-property :version, kind_of: [String, Symbol], default: :latest
-property :channel, kind_of: Symbol, default: :stable, equal_to: [:stable, :current, :unstable]
+property :version, [String, Symbol], default: :latest
+property :channel, Symbol, default: :stable, equal_to: [:stable, :current, :unstable]
 
 # Install package from local file
-property :package_source, kind_of: String
+property :package_source, String
 
 # Set the *-ctl command to use when doing reconfigure
-property :ctl_command, kind_of: String
+property :ctl_command, String
 
 # Package resources used on rhel and debian platforms
-property :options, kind_of: String
-property :timeout, kind_of: [Integer, String, NilClass]
```

Comment:
I'm debating adding `nil` back into the type array now since I'm sure sure if this will break cookbooks that may be setting `timeout` to `nil`.